tools/building.py

```diff
@@ -1432,7 +1432,7 @@ def emit_wasm_source_map(wasm_file, map_file, final_wasm):
   # source file paths must be relative to the location of the map (which is
   # emitted alongside the wasm)
   base_path = os.path.dirname(os.path.abspath(final_wasm))
-  sourcemap_cmd = [PYTHON, path_from_root('tools/wasm-sourcemap.py'),
+  sourcemap_cmd = [sys.executable, '-E', path_from_root('tools/wasm-sourcemap.py'),
```

Comment:
Reading the python docs, `-E` seems to ignore environment variables. I'm not sure how that relates to the explanation in the PR description about `-E` here..?

Reply:
I will revert this since its going to land in  #16736 anyway.